chain/chain/src/validate.rs

```diff
@@ -190,10 +189,13 @@ pub fn validate_chunk_with_chunk_extra_and_receipts_root(
     Ok(())
 }
 
+#[derive(BorshSerialize)]
+struct TransactionReceiptRef<'a>(&'a [SignedTransaction], &'a [Receipt]);
```

Comment:
Any way you could add a test to verify that this is ~equivalent to `TransactionReceipt` for the purposes of encoding? Maybe the implementation of `BorshSerialize` for that could defer to `TransactionReceiptRef`?

Reply:
Sure, I'll add a test